dbms/src/Disks/DiskLocal.cpp

```diff
@@ -95,6 +106,19 @@ void DiskLocal::createDirectories(const String & path)
     Poco::File(disk_path + path).createDirectories();
 }
 
+void DiskLocal::clearDirectory(const String & path)
+{
+    std::vector<Poco::File> files;
+    Poco::File(disk_path + path).list(files);
+    for (auto & file : files)
+        file.remove(true);
+}
+
+void DiskLocal::moveDirectory(const String & from_path, const String & to_path)
```

Comment:
It's inconsistent with:

MergeTreeData.cpp, line 1287:
``` 
       disk->moveFile(relative_data_path, new_table_path);
```

Please clarify.

Reply:
It's a mistake. I've replaced moveFile() with moveDirectory() in MergeTreeData.cpp.